src/markdown-pages/about_Galasa.md

```diff
@@ -3,19 +3,41 @@ path: "/about"
 title: "About Galasa"
 ---
 
-Galasa is different from other test tools. It is a deep integration test framework, enabling you to test applications that interact with z/OS (including CICS applications) without stubbing or mocking. 
+Galasa is different from other test tools; enabling deep integration testing across platforms and technologies within a DevOps pipeline, Galasa supports repeatable, reliable, agile testing at scale across your enterprise.
 
-Galasa tests are not limited to testing mainframe-specific applications - they can extend to encompass entire families of hybrid cloud applications. Most manual tests can be automated with Galasa, helping you to deliver changes to your core applications quickly and successfully. 
+### Platform integration
+The Galasa framework is built with a native knowledge of z/OS and hybrid multi-cloud, enabling end-to-end testing of applications that run on different platforms without stubbing or mocking. 
+
+### Technology integration
+Galasa is fully extensible, supporting integration across technologies by using an API that interacts with multiple test tools, including JMeter, Selenium, JCL and 3270 screens and more. 
+
+### Pipeline integration
+Galasa provides a REST endpoint that can be called from any CI/CD pipeline, for example, a Jenkins pipeline. Running Galasa as part of your pipeline enables fast, reliable testing at scale - Galasa can run thousands of tests in parallel. 
+
+### Enterprise integration
+Galasa is not simply about providing the ability to write a test and run it in automation.  Galasa's test catalog enables you to understand and manage your test corpus, making it easy to schedule the right tests to run at the right time. Test results and artifacts are stored in a single repository, making reporting and problem determination simple when combined with native Kibana and Grafana dashboards. 

```

Comment:
corpus or catalog?

Reply:
changed it to read tests rather than test corpus